specification/source/cdl.rst

```diff
@@ -1688,118 +1688,97 @@ Therefore, tools that process CDL can infer the following information:
+   annotation (__Buildings(semantic(<semantic information>)))
+   
+* The top-level block declaration shall house the ``metadataLanguageDefinition`` within the  ``semantic`` section. ``metadataLanguageDefinition`` will be used to define the different types of tags that are used throughout the CDL block and relevant information about the same. It will follow the following syntax:
 
 .. code-block:: modelica
 
-   annotation :
-     annotation "(" [annotations ","]
-        __cdl "(" [ __cdl_annotation ] ")" ["," annotations] ")"
+   annotation (__cdl(semantic(metadataLanguageDefinition="<metadataLanguage> [version] [format]" "<description or URL>")))
+   annotation (__Buildings(semantic(metadataLanguageDefinition="<metadataLanguage> [version] [format]" "<description or URL>")))
 
-where ``__cdl__annotation`` is the annotation for CDL.
-
-For Brick, the ``__cdl_annotation`` is
+* ``metadataLanguageDefinition`` could be semantic data standards such as  Brick, Project-Haystack, ASHRAE 223p etc. or it could just be a natural language description. The optional ``version`` represents the version of the metadataLangauge used in a particular sequence and the optional ``format``  represents the format this will be expressed in.  The format will be expressed using  `MIME types <https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types>`_. Examples:
 
 .. code-block:: modelica
 
-   brick_annotation:
-      brick "(" RDF ")"
+  annotation (__cdl(semantic(
+    metadataLanguageDefinition="Brick 1.3 text/turtle" "https://brickschema.org/ontology/1.3",
+    metadataLanguageDefinition="Project-Haystack 3.9.12 application/ld+json" "https://project-haystack.org/",
+    metadataLanguageDefinition="Text" "string in English language"
+  )));
 
-where ``RDF`` is the RDF 1.1 Turtle
-(https://www.w3.org/TR/turtle/) specification of the Brick object.
+* The semantic information itself will be included as a metadataLanguage/metadata pair within the annotation using the following syntax:
 
-[Note that, for example
-for a controller with two output signals :math:`y_1` and :math:`y_2`,
-Brick seems to have no means to specify
-that :math:`y_1` ``controls`` a fan speed and :math:`y_2` ``controls``
-a heating valve, where ``controls`` is the Brick relationship.
-Therefore, we allow the ``brick_annotation`` to only be at the
-block level, but not at the level of instances of input or output
-connectors.
+.. code-block:: modelica
+
+  annotation (__cdl(semantic(metadataLanguage="<metadataLanguage> [version] [format]" "<metadata>")));
+  annotation (__Buildings(semantic(metadataLanguage="<metadataLanguage> [version] [format]" "<metadata>")));
 
-For example, the Brick specification
+* Annotations attached to the top-level block can also be used to define block level information about the semantic data. A top-level block is one that encompasses a particular sequence or a model. For example, in Brick 1.3, it can be used to define prefixes. Example:
```

Comment:
What does "attached to the top-level mean"? Better write something like "Semantic information declared in the block-level annotation ...".
The use of "top-level" is problematic as a block may be at a top-level, but if used within another composite block, it is no longer at the top-level. 
Also, a block cannot encompass (contain?) a model as this is not valid Modelica.